pkg/controller/plan/adapter/converter.go

```diff
@@ -134,7 +139,9 @@ func createConvertJob(pvc *v1.PersistentVolumeClaim, srcFormat, dstFormat string
 						SeccompProfile: &v1.SeccompProfile{
 							Type: v1.SeccompProfileTypeRuntimeDefault,
 						},
-						FSGroup: ptr.To(int64(107)),
+						RunAsNonRoot: ptr.To(true),
+						RunAsUser:    ptr.To[int64](1000850000),
+						FSGroup:      ptr.To[int64](1000850000),
```

Comment:
that's temporary, right?

Reply:
we can use it... we can add are own range if it makes a difference, there's no reason in this context to specifically use the qemu numbers

Reply:
this looks fishy. pretty strange that we could make other pods work and now we need this

Reply:
we don't have forklift's securitycontext in this case because the pod is create from a job